src/shell/dotnet/test/Shell.Tests/Shell.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(TargetFrameworkVersion)-windows</TargetFramework>
```

Comment:
If we really need to hardcode this specific version, please keep a central parameter for it and use it in both the shell and the shel.tests project.